src/views/Home/index.scss

```diff
@@ -1,8 +1,18 @@
+@use "../../stylesheets/variables";
+
 .home__container {
   display: flex;
   flex-direction: column;
 
   max-width: 1000px;
   margin: 0 auto;
   padding: 0 2em;
-}
\ No newline at end of file
+
+  .app--light {
+    @include variables.light-background;
+  }
+
+  .app--dark {
+    @include variables.dark-background;
+  }
```

Comment:
These lines do not do anything

Reply:
Oh yeah, I added them while I was testing something and forgot to remove.